docs/docs/repository.md

```diff
@@ -379,6 +379,29 @@ response.result match {
 
 The `result` on the right is the corresponding [List[Release]][repository-scala].
 
+### The latest release
```

Comment:
can you create links at the top: https://github.com/47degrees/github4s/pull/493/files#diff-361b3d5e27b7fb71bacbd4f15457901eR26